src/common/Version.hpp

```diff
@@ -29,6 +29,7 @@ class Version
     const QString &dateOfBuild() const;
     const QString &fullVersion() const;
     const bool &isSupportedOS() const;
+    const bool isFlatpak() const;
```

Comment:
```suggestion
    bool isFlatpak() const;
```

Since we return a copy anyway, this should return a `bool` instead of a `const bool`. (Also see [this note](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#note-57) in the C++ core guidelines)
(Remember to change the function definition in the source file as well)